NuRadioReco/detector/detector.py

```diff
@@ -94,18 +99,19 @@ def Detector(*args, **kwargs):
         if len(args) >= 4:
             assume_inf = args[3]
         else:
-            assume_inf = kwargs.pop("assume_inf", True)
+            assume_inf = kwargs.pop("assume_inf", None)
```

Comment:
is that necessary here? If both classes have the same default for this argument?

Reply:
True, but this change is to accommodate the default values from NuradioRecoio.py . If the value for assume_inf is not in the .nur file, it defaults to None. Rather then setting a default value there as well (making it hardcoded in that place else as well), I opted to change this here.

Reply:
okay